pyscal/pyscalcli.py

```diff
@@ -149,6 +150,31 @@ def get_parser() -> argparse.ArgumentParser:
             "Implicit for gas-water input."
         ),
     )
+
+    # Plotting arguments
+    parser.add_argument(
+        "--plot",
+        action="store_true",
+        default=False,
+        help=("Make and save relative permeability figures."),
+    )
+    parser.add_argument(
+        "--pc",
+        action="store_true",
+        default=False,
+        help=("Make and save capillary pressure figures."),
+    )
```

Comment:
Will it be better if we make it more explicit that it's referring to plotting Pc figure? Something like `plotpc`